Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -485,3 +485,121 @@ public extension TransposedConv2D {
+        self.padding = padding
+    }
+
+    /// Returns the output obtained from applying the layer to the given input.
+    ///
+    /// - Parameter input: The input to the layer.
+    /// - Returns: The output.
+    @differentiable
+    public func callAsFunction(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
+        return activation(depthwiseConv2D(input, depthwise_kernel: depthwise_kernel,
+                                 strides: (1, strides.0, strides.1, 1),
+                                 padding: padding) + bias)
+    }
+}
+
+public extension DepthwiseConv2D {
+    /// Creates a `depthwiseConv2D` layer with the specified depthwise_kernel shape, strides, padding, and
+    /// element-wise activation function. The depthwise_kernel tensor is initialized using Glorot uniform
+    /// initialization with the specified generator. The bias vector is initialized with zeros.
+    ///
+    /// - Parameters:
+    ///   - depthwiseKernelShape: The shape of the 4-D convolution kernel.
+    ///   - strides: The strides of the sliding window for spatial/spatio-temporal dimensions.
+    ///   - padding: The padding algorithm for convolution.
+    ///   - activation: The element-wise activation function.
+    ///   - generator: The random number generator for initialization.
+    ///
+    /// - Note: Use `init(filterShape:strides:padding:activation:seed:)` for faster random
+    ///   initialization.
+    init<G: RandomNumberGenerator>(
+        depthwiseKernelShape: (Int, Int, Int, Int),
+        strides: (Int, Int) = (1, 1),
+        padding: Padding = .valid,
+        activation: @escaping Activation = identity,
+        generator: inout G
+    ) {
+        let depthwiseKernelTensorShape = TensorShape([
+            depthwiseKernelShape.0, depthwiseKernelShape.1, depthwiseKernelShape.2,
+            depthwiseKernelShape.3])
+        self.init(
+            depthwise_kernel: Tensor(glorotUniform: depthwiseKernelTensorShape, generator: &generator),
+            bias: Tensor(zeros: TensorShape([depthwiseKernelShape.3])),
+            activation: activation,
+            strides: strides,
+            padding: padding)
+    }
+}
+
+public extension DepthwiseConv2D {
+    /// Creates a `depthwiseConv2D` layer with the specified depthwise_kernel shape, strides, padding, and
+    /// element-wise activation function. The depthwise_kernel tensor is initialized using Glorot uniform
+    /// initialization with the specified seed. The bias vector is initialized with zeros.
+    ///
+    /// - Parameters:
+    ///   - depthwiseKernelShape: The shape of the 4-D convolution kernel.
+    ///   - strides: The strides of the sliding window for spatial/spatio-temporal dimensions.
+    ///   - padding: The padding algorithm for convolution.
+    ///   - activation: The element-wise activation function.
+    ///   - seed: The random seed for initialization. The default value is random.
+    init(
+        depthwiseKernelShape: (Int, Int, Int, Int),
```

Comment:
`depthwiseKernelShape` -> `kernelShape`, since the word "depthwise" is repeating the type information about the layer.